app/organization/invite/route.js

```diff
@@ -43,6 +44,10 @@ export default Ember.Route.extend({
 
         let newInvite = this.store.createRecord('invitation');
         this.controller.set('model', newInvite);
+      }).then(() => {
+        return organization.get('invitations').then((invites) => {
+          invites.pushObject(invitation);
+        });
```

Comment:
I think this may be better achieved by having an `organization` in the `options` when creating an invitation. In the `model` hook.